ui/utils.py

```diff
@@ -221,7 +221,8 @@ def fit_image_to_screen(image, o):
 
     image_width, image_height = image.size
     if o.height > image_height and o.width > image.width: # Checks if the screen dimensions are equal to the image size
-        if o.height/image_height < o.width/image_width: # Checks which side will is bigger in proportion to the image size
+	logger.info("Using the resize script")
```

Comment:
won't this error out? you have some interesting indentation there. Also, no need to add logging for the call itself in the function

Reply:
That's why I like automated tests - in general, even just running the `ui/printer.py` would've caught it (because it's a `SyntaxError`). However, no problems with rushed commits, if you have to go, feel free to commit things as-is =)

Reply:
Okay thanks! Thought you would rather have the code now than in a days time =D.

Reply:
Yep, that's what is more important to me =) Thank you!